src/content/en/2020/css.md

```diff
@@ -115,22 +126,40 @@ Only a few of the top classes were presentational, with most of those being eith
 
 Despite IDs being discouraged these days in some circles due to their much higher specificity, most websites still use them, albeit sparingly. Fewer than half of pages used more than one ID in any of their selectors (had a max specificity of (1,x,y) or less) and nearly all had a median specificity that did not include IDs (0,x,y).
 
+{# TODO Authors—Can you explain this (1,x,y)/(0,x,y) syntax more? #}
+
 But what are these IDs used for? It turns out that the most popular IDs are structural: `#content`, `#footer`, `#header`, `#main`, despite [corresponding HTML elements](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#HTML_layout_elements_in_more_detail).
 
+{# TODO Authors—This sentence just ends. Do you mean "despite corresponding HTML elements existing that could be used as selectors while also improving the semnatic markup"? #}
+
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
Yes, that is a better way to phrase it.

Reply:
```suggestion
But what are these IDs used for? It turns out that the most popular IDs are structural: `#content`, `#footer`, `#header`, `#main`, despite [corresponding HTML elements](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#HTML_layout_elements_in_more_detail) existing that could be used as selectors while also improving the semantic markup.

```